packages/dos-donts-block/src/DoDontItem.tsx

```diff
@@ -270,6 +270,7 @@ export const DoDontItem = React.forwardRef<HTMLDivElement, DoDontItemProps>(
                                         {
                                             ...designTokens?.heading3,
                                             marginBottom: 0,
+                                            marginTop: 0,
```

Comment:
I still have an issue with the top margin locally. If you also add it to line 263 like you did with the marginBottom it works on my side.